### PR TITLE
Fix types packaging

### LINKED ISSUE #7: Error when using in a NextJS / Typescript project
Hello,

I get this error when trying to use this lib in a TS project:
```
  There are types at '(...)/node_modules/overpass-ql-ts/lib/types-bundle.d.ts', but this result could not be resolved when respecting package.json "exports". The 'overpass-ql-ts' library may need to update its package.json or typings.
```

### PR DESCRIPTION
**Pull request by Djiit**
Fixes #7

### CODE REVIEW

**Comment by JuanCouste** [comment]
Does it need to be written like types-bundle **.mjs** ?
Could it not be written like types-bundle **.d.ts** ?

File: package.json [hunk 1]
```diff
@@ -68,6 +68,7 @@
 			"module": "./lib/esm-bundle.mjs",
 			"import": "./lib/esm-bundle.mjs",
 			"require": "./lib/cjs-bundle.cjs",
+			"types": "./lib/types-bundle.mjs",
```
Comment (Djiit) [inline package.json:71]:
```suggestion
			"types": "./lib/types-bundle.d.ts"
```

**Comment by Djiit** [comment]
Oops yes, I fixed the naming so it matches the existing files. Thanks for noticing 🙏

File: package.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
 			"module": "./lib/esm-bundle.mjs",
 			"import": "./lib/esm-bundle.mjs",
 			"require": "./lib/cjs-bundle.cjs",
-			"types": "./lib/types-bundle.mjs",
+			"types": "./lib/types-bundle.d.ts"
```
Comment (JuanCouste) [inline package.json:71]: I would love to have a comma at the end of this line, npm really does not like it 😂

**Comment by JuanCouste** [comment]
Could you check if merging your changes from [Djiit:patch-1](https://github.com/Djiit/overpass-ql-ts/tree/patch-1) to [Djiit:master](https://github.com/Djiit/overpass-ql-ts/tree/master) triggers the test action?
It would be interesting if i can see those results from a pr

File: package.json [hunk 3]
```diff
@@ -68,6 +68,7 @@
 			"module": "./lib/esm-bundle.mjs",
 			"import": "./lib/esm-bundle.mjs",
 			"require": "./lib/cjs-bundle.cjs",
+			"types": "./lib/types-bundle.d.ts"
```
Comment (Djiit) [inline package.json:71]:
```suggestion
			"types": "./lib/types-bundle.d.ts",
```

**Comment by Djiit** [comment]
Looks like it's running here yes: https://github.com/Djiit/overpass-ql-ts/actions

**Pull request merged**